module-system/sov-state/src/prover_storage.rs

```diff
@@ -191,25 +185,25 @@ impl<S: MerkleProofSpec> Storage for ProverStorage<S> {
         proof.verify(state_root, key_hash, value.as_ref().map(|v| v.value()))?;
         Ok((key, value))
     }
-}
 
-impl<S: MerkleProofSpec> NativeStorage for ProverStorage<S> {
-    fn get_with_proof(
-        &self,
-        key: StorageKey,
-        _witness: &Self::Witness,
-    ) -> StorageProof<Self::Proof> {
+    // Based on assumption `validate_and_commit` increments version.
+    fn is_empty(&self) -> bool {
+        self.db.get_next_version() <= 1
+    }
+
+    #[cfg(feature = "native")]
```

Comment:
I believe ` #[cfg(feature = "native")]` is not needed here.

Reply:
I followed approach of

```
    #[cfg(feature = "native")]
    fn get_accessory(&self, key: &StorageKey) -> Option<StorageValue> {
    ```